docs/breaking-changes.asciidoc

```diff
@@ -1,47 +1,49 @@
 [[breaking_changes]]
-=== Breaking changes from 6.x
+=== Breaking changes from 7.x
 
-[discrete]
-==== E_USER_DEPRECATED notice when using deprecated parameters
-
-Starting from elasticsearch-php 7.4.0, we generate a PHP 
-https://www.php.net/manual/en/errorfunc.constants.php[E_USER_DEPRECATED] notice 
-every time you use a deprecated parameters for {es}. We decided to add this 
-notice to facilitate the code refactoring with the new API specification of {es} 
-(for example the usage of typeless APIs, see below).
-
-From 7.4.1, we mitigated the usage of E_USER_DEPRICATED warning by using the `@` 
-https://www.php.net/manual/en/language.operators.errorcontrol.php[operator]. The 
-operator suppresses the error message, however, it is still possible to catch it 
-by using a custom error handler:
-
-[source,php]
-----
-set_error_handler(function ($errno, $errstr) {
-    var_dump($errstr);
-}, E_USER_DEPRECATED);
-
-@trigger_error('Deprecation message here', E_USER_DEPRECATED);
-----
+This major release is a complete new PHP client for {es}. We build it from scratch!
+We tried to reduce the BC breaks as much as possible but there are some (big) differences: 
 
 [discrete]
-==== Moving from types to typeless APIs in {es} 7.0
+==== Architectural changes:
 
-{es} 7.0 deprecated APIs that accept types, introduced new typeless APIs, and 
-removed support for the _default_ mapping. Read 
-https://www.elastic.co/blog/moving-from-types-to-typeless-apis-in-elasticsearch-7-0[this]
-blog post for more information.
+- we changed the namespace, now everything is under `Elastic\Elasticsearch`
+- we used the [elastic-transport-php](https://github.com/elastic/elastic-transport-php) library for HTTP communications;
+- we changed the `Exception` model, using the namespace `Elastic\Elasticsearch\Exception`. All the exceptions extends the
+  `ElasticsearchException` interface, as in 7.x
+- we changed the response type of each endpoints using an [Elasticsearch](src/Response/Elasticsearch.php) response class.
```

Comment:
I changed the link to an absolute one, please check if I used the proper one. I think it's from the readme file but it won't work on the rendered HTML page as a relative link. Also MD --> Asciidoc.

```suggestion
- we changed the response type of each endpoint using an https://github.com/elastic/elasticsearch-php/blob/master/src/Response/Elasticsearch.php[Elasticsearch] response class.
```